seabird.proto

```diff
@@ -4,168 +4,163 @@ option go_package = ".;pb";
 
 package seabird;
 
-// Identity is included in every message that requires auth. One AuthMethod at a
-// time is supported. For now, the only supported method is a token. These need
-// to be created manually and distributed to any clients.
-message Identity {
-  oneof AuthMethod {
-    string token = 1;
-  }
-}
-
-service Seabird {
-  rpc StreamEvents(StreamEventsRequest) returns (stream Event);
+import "common.proto";
 
-  // Chat actions
-  rpc SendMessage(SendMessageRequest) returns (SendMessageResponse);
-  rpc JoinChannel(JoinChannelRequest) returns (JoinChannelResponse);
-  rpc LeaveChannel(LeaveChannelRequest) returns (LeaveChannelResponse);
+// Note that the IDs exposed by the Service, User, and Channel are compound IDs
```

Comment:
Details on what `compound` means? Maybe an example? I would either say "they're compound and here's an example" or say "do not depend on the structure of these IDs", otherwise plugins might depend on the structure without it being guaranteed to stay the same.

Reply:
Also which service will route these? If you're talking about routing to the backend, does that mean `ChannelSource` is actually `ChannelDestination`? I don't think you're talking about plugin routing as that's not really supported.

Reply:
I changed "compound" to "opaque" and clarified that core will route relevant items to the proper backend